24/12/21 (14:59:35) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1137, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
24/12/21 (14:59:42) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1137, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
24/12/21 (14:59:52) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1137, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
24/12/21 (15:00:07) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1137, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
24/12/21 (15:00:24) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1137, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
24/12/21 (15:00:30) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1137, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
24/12/21 (15:20:46) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1137, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
24/12/21 (15:29:28) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1137, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
24/12/21 (15:32:04) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1137, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
24/12/21 (15:39:28) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1137, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
24/12/21 (15:41:30) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1137, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
24/12/21 (16:24:21) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1137, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
24/12/21 (19:26:14) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1137, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
26/12/21 (11:15:10) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
27/12/21 (11:41:35) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
27/12/21 (16:40:34) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
27/12/21 (16:40:50) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
27/12/21 (16:40:54) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
27/12/21 (16:41:10) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
27/12/21 (16:41:25) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
27/12/21 (16:42:02) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
27/12/21 (16:42:24) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
27/12/21 (16:42:42) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
27/12/21 (16:42:53) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
27/12/21 (16:46:10) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
27/12/21 (16:47:25) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
27/12/21 (16:47:30) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
27/12/21 (16:49:11) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
27/12/21 (16:49:23) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
27/12/21 (16:49:42) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
28/12/21 (14:34:54) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
29/12/21 (09:29:54) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
29/12/21 (09:32:19) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
29/12/21 (11:25:17) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
29/12/21 (11:43:28) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
29/12/21 (11:43:29) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
29/12/21 (13:06:22) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
30/12/21 (16:10:50) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
31/12/21 (13:08:06) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
31/12/21 (14:23:13) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
06/01/22 (10:58:45) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
06/01/22 (15:04:49) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
09/01/22 (19:20:19) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
10/01/22 (19:50:44) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
10/01/22 (19:50:50) | ERROR: The read operation timed out
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 5029, in _checker
    with urllib.request.urlopen("https://github.com/aristocratos/bpytop/raw/master/bpytop.py", timeout=5) as source: # type: ignore
  File "/usr/lib/python3.10/urllib/request.py", line 216, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.10/urllib/request.py", line 525, in open
    response = meth(req, response)
  File "/usr/lib/python3.10/urllib/request.py", line 634, in http_response
    response = self.parent.error(
  File "/usr/lib/python3.10/urllib/request.py", line 557, in error
    result = self._call_chain(*args)
  File "/usr/lib/python3.10/urllib/request.py", line 496, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.10/urllib/request.py", line 749, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/lib/python3.10/urllib/request.py", line 519, in open
    response = self._open(req, data)
  File "/usr/lib/python3.10/urllib/request.py", line 536, in _open
    result = self._call_chain(self.handle_open, protocol, protocol +
  File "/usr/lib/python3.10/urllib/request.py", line 496, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.10/urllib/request.py", line 1391, in https_open
    return self.do_open(http.client.HTTPSConnection, req,
  File "/usr/lib/python3.10/urllib/request.py", line 1352, in do_open
    r = h.getresponse()
  File "/usr/lib/python3.10/http/client.py", line 1374, in getresponse
    response.begin()
  File "/usr/lib/python3.10/http/client.py", line 318, in begin
    version, status, reason = self._read_status()
  File "/usr/lib/python3.10/http/client.py", line 279, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "/usr/lib/python3.10/socket.py", line 705, in readinto
    return self._sock.recv_into(b)
  File "/usr/lib/python3.10/ssl.py", line 1273, in recv_into
    return self.read(nbytes, buffer)
  File "/usr/lib/python3.10/ssl.py", line 1129, in read
    return self._sslobj.read(len, buffer)
TimeoutError: The read operation timed out
10/01/22 (20:05:24) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
11/01/22 (10:14:34) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
11/01/22 (11:17:57) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
11/01/22 (11:34:19) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
11/01/22 (13:07:21) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
11/01/22 (13:07:21) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
11/01/22 (13:07:21) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
11/01/22 (13:12:06) | ERROR: Incorrectly formatted hexadecimal rgb string: #30303000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #30303000
11/01/22 (13:24:24) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
11/01/22 (19:41:30) | ERROR: The read operation timed out
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 5029, in _checker
    with urllib.request.urlopen("https://github.com/aristocratos/bpytop/raw/master/bpytop.py", timeout=5) as source: # type: ignore
  File "/usr/lib/python3.10/urllib/request.py", line 216, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.10/urllib/request.py", line 525, in open
    response = meth(req, response)
  File "/usr/lib/python3.10/urllib/request.py", line 634, in http_response
    response = self.parent.error(
  File "/usr/lib/python3.10/urllib/request.py", line 557, in error
    result = self._call_chain(*args)
  File "/usr/lib/python3.10/urllib/request.py", line 496, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.10/urllib/request.py", line 749, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/lib/python3.10/urllib/request.py", line 519, in open
    response = self._open(req, data)
  File "/usr/lib/python3.10/urllib/request.py", line 536, in _open
    result = self._call_chain(self.handle_open, protocol, protocol +
  File "/usr/lib/python3.10/urllib/request.py", line 496, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.10/urllib/request.py", line 1391, in https_open
    return self.do_open(http.client.HTTPSConnection, req,
  File "/usr/lib/python3.10/urllib/request.py", line 1352, in do_open
    r = h.getresponse()
  File "/usr/lib/python3.10/http/client.py", line 1374, in getresponse
    response.begin()
  File "/usr/lib/python3.10/http/client.py", line 318, in begin
    version, status, reason = self._read_status()
  File "/usr/lib/python3.10/http/client.py", line 279, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "/usr/lib/python3.10/socket.py", line 705, in readinto
    return self._sock.recv_into(b)
  File "/usr/lib/python3.10/ssl.py", line 1273, in recv_into
    return self.read(nbytes, buffer)
  File "/usr/lib/python3.10/ssl.py", line 1129, in read
    return self._sslobj.read(len, buffer)
TimeoutError: The read operation timed out
23/01/22 (18:49:05) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
23/01/22 (18:49:12) | ERROR: Incorrectly formatted hexadecimal rgb string: #00000000
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1157, in __init__
    raise ValueError(f'Incorrectly formatted hexadecimal rgb string: {self.hexa}')
ValueError: Incorrectly formatted hexadecimal rgb string: #00000000
